tests/common/ctx.rs

```diff
@@ -40,15 +63,22 @@ pub async fn ctx() -> Option<Context> {
 
 #[macro_export]
 macro_rules! setup {
-    () => {{
+    () => {
+        setup!("mainnet")
+    };
+    ($e:expr) => {{
         let run: bool = std::env::var("BEERUS_TEST_RUN")
             .ok()
             .map(|value| &value == "1")
             .unwrap_or_default();
         if !run {
             return Ok(());
         }
-        if let Some(ctx) = common::ctx::ctx().await {
+        let context = match $e {
+            "sepolia" => common::ctx::ctx_sepolia().await,
+            _ => common::ctx::ctx_mainnet().await,
```

Comment:
This allows for funny and non-obvious things like `setup!("lol")` to run against mainnet. Making it more strict maybe?

Reply:
Updated